Example/src/MeasureExample.tsx

```diff
@@ -104,18 +122,28 @@ function Section({ title, children, height, contentHeight, z, show }) {
         show={show}
       />
       <View>
-        {React.Children.map(children, (element, idx) => {
-          return React.cloneElement(element, { ref: aref });
-        })}
+        {React.Children.map(children, (element) =>
+          React.cloneElement(element as ReactElement, { ref: aref })
```

Comment:
Can't you pass a type to `React.Children`?

Reply:
You can pass generic types to `.map` but I'm not sure how to type this.

```
Argument of type 'ReactNode' is not assignable to parameter of type 'ReactElement<any, string | ((props: any) => ReactElement<any, any> | null) | (new (props: any) => Component<any, any, any>)> | ReactElement<any, string | ... 1 more ... | (new (props: any) => Component<...>)>[]'.
  Type 'undefined' is not assignable to type 'ReactElement<any, string | ((props: any) => ReactElement<any, any> | null) | (new (props: any) => Component<any, any, any>)> | ReactElement<any, string | ... 1 more ... | (new (props: any) => Component<...>)>[]'.
```

Reply:
I think we can leave it as it is then.